### PR TITLE
Update handlers_test.go

### PR DESCRIPTION
**Pull request by jofmi**
# Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Description
Add test to check if balances are updated after order is verified

# Checklist:

- [ ] I have commented my code (or ChatGPT did), particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings, neither in my IDE nor in my browser
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: app/handlers/handlers_test.go [hunk 1]
```diff
@@ -251,13 +250,12 @@ func TestOrders(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
+	log.Info(senderAccount, "senderAccount")
+	log.Info(receiverAccount, "receiverAccount")
 	require.Equal(t, senderAccount.Balance, -98910)
-	require.Equal(t, receiverAccount.Balance, 98910)
+	require.Equal(t, receiverAccount.Balance, 60165)
```
Comment (lebe1) [inline app/handlers/handlers_test.go:256]: Weird balance check, which means another payment must have been done before

**Comment by lebe1** [comment]
Hi @jofmi, I just pushed a commit to fix a testing error, which lead to a different testing in test_payments(). Do you think we might have a similar problem here due to parallel running tests asking the test_DB how many payments have been done, which can differ depending what tests ran before?
Also I fixed test_orders with a complete different balance check as you can see above in my review comment...

**Comment by jofmi** [comment]
> Hi @jofmi, I just pushed a commit to fix a testing error, which lead to a different testing in test_payments(). Do you think we might have a similar problem here due to parallel running tests asking the test_DB how many payments have been done, which can differ depending what tests ran before? 

Yes, the tests are passing locally for me, so it must be the same issue with the CI ...

File: app/handlers/handlers_test.go (hunk 1 above)
Comment (jofmi) [reply app/handlers/handlers_test.go]: I have simplified the test numbers in the last commit.

So the thought process is the following:

- There's a newspaper for the price of 20 with a license for the price of 3
- In the test, the user buys 2 newspapers, giving them a balance of -40
- The vendor has to pay for two licenses, giving them a balance of -6, and then gets the money for the newspapers, leaving them with a balance of 34
- The missing 6 land in the balance of the organisation which receives the money for the license

**Comment by jofmi** [comment]
Since we now know that the problem is not about parallelism, I fixed the error by deleting the data from the test at the end of the test

File: app/handlers/handlers_test.go [hunk 2]
```diff
@@ -221,20 +222,49 @@ func TestOrders(t *testing.T) {
...
+	senderAccount, err = database.Db.GetAccountByType("UserAnon")
+	if err != nil {
+		t.Error(err)
+	}
+	receiverAccount, err = database.Db.GetAccountByVendorID(vendorIDInt)
+	if err != nil {
+		t.Error(err)
+	}
+	log.Info(senderAccount, "senderAccount")
+	log.Info(receiverAccount, "receiverAccount")
```
Comment (lebe1) [inline app/handlers/handlers_test.go:255]: Pls remove these logs

Comment (jofmi) [reply app/handlers/handlers_test.go]: done!

**Review by lebe1 — APPROVED** [review]

**Pull request merged**